app/models/wiki_content/revision.rb

```diff
@@ -68,6 +68,15 @@ def plagiarism_report_link
     "/recent-activity/plagiarism/report?ithenticate_id=#{ithenticate_id}"
   end
 
+  def wikidata_ref_tags
+    features['feature.len(<datasource.wikidatawiki.revision.references>)']
+  end
+
+  def wikidata_ref_tags_previous
+    return 0 if new_article
+    features_previous['feature.len(<datasource.wikidatawiki.revision.references>)']
```

Comment:
I think this would work better if you incorporated it into `ref_tags` instead of having a parallel method dedicated to wikidata.